ui/apps/platform/src/Containers/Clusters/cluster.helpers.tsx

```diff
@@ -187,13 +188,17 @@ const upgradeStates: UpgradeStates = {
         type: 'current',
     },
     MANUAL_UPGRADE_REQUIRED: {
-        displayValue: 'Manual upgrade required',
+        displayValue: `Version not managed by ${getProductBranding().reportName}`,
```

Comment:
Maybe `Sensor is not managed by ${getProductBranding().reportName}.` if we know this will only appear for sensor.